File: python/svix/models/auto_config_subscription_out.py
# this file is @generated
import typing as t
from datetime import datetime

from .common import BaseModel
from .status import Status


class AutoConfigSubscriptionOut(BaseModel):
    created_at: datetime

    token_censored: str

    id: str
    """The AutoConfigSubscription's ID."""

    endp_id: t.Optional[str] = None
    """The Endpoint's ID."""

    dest_id: t.Optional[str] = None
    """The StreamSink's ID."""

    status: Status

File: kotlin/lib/src/main/kotlin/models/AutoConfigSubscriptionOut.kt
// This file is @generated
package com.svix.kotlin.models

import kotlin.time.Instant
import kotlinx.serialization.Serializable

@Serializable
data class AutoConfigSubscriptionOut(
    val createdAt: Instant,
    /** The censored AutoConfig token */
    val tokenCensored: String,
    /** The AutoConfigSubscription's ID. */
    val id: String,
    /** The Endpoint's ID. */
    val endpId: String? = null,
    /** The StreamSink's ID. */
    val destId: String? = null,
    val status: Status,
)
